test/utility/genome_size_tests.py

```diff
@@ -33,51 +32,69 @@ def setUpClass(cls):
             cls.cfg[nameval[0]] = nameval[1]
         cls.wsURL = cls.cfg['workspace-url']
         cls.wsClient = workspaceService(cls.wsURL, token=token)
+        cls.dfuClient = DataFileUtil(os.environ['SDK_CALLBACK_URL'], token=token, service_ver='dev')
         cls.serviceImpl = GenomeFileUtil(cls.cfg)
-        cls.token = token
+
+        suffix = int(time.time() * 1000)
+        cls.wsName = "test_GenomeFileUtil_" + str(suffix)
+        cls.wsClient.create_workspace({'workspace': cls.wsName})
+        cls.wsID = cls.dfuClient.ws_name_to_id(cls.wsName)
```

Comment:
```suggestion
        cls.wsID = cls.wsClient.create_workspace({'workspace': cls.wsName})[0]
```
pretty sure that's the right syntax, but you might need to fix it if not. create-workspace def returns the wsid

Reply:
👍